tools/src/bin/crossref.rs

```diff
@@ -98,7 +99,7 @@ fn process_analysis_target(
     // see how it works out.
```

Comment:
Maybe this comment should now be something like:

The C++ indexer now should properly emit context for all FILE symbols of the file that they are found in, but we're retaining this for older C++ code and because it might be useful for other indexers that emit FILE symbols.